tests/unit/ui/components/TestDismissibleTextForm.js

```diff
@@ -107,20 +107,6 @@ describe(__filename, () => {
     sinon.assert.called(onDismiss);
   });
 
-  it('clears the text when dismissing a textarea', () => {
```

Comment:
This behaviour didn't seem needed for anything and interfered with cancelling an abuse report but then opening it again–it would clear the contents of the textarea. This didn't seem there for a reason and was bad UX (as "cancel" might've been clicked by mistake, undoing a lot of effort in typing the abuse report).